services/openvpn/service/manager.go

```diff
@@ -149,6 +151,11 @@ func (m *Manager) Serve(providerID identity.Identity) (err error) {
 		}
 	}()
 
+	err = m.shaper.Start(m.vpnServer.DeviceName())
+	if err != nil {
+		log.Error("Could not start traffic shaper: ", err)
```

Comment:
Is it safe to call m.vpnServer.Wait() if you have error here?

Reply:
Yes